platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -484,14 +484,21 @@ class MeasurementService {
     return newMeasurement.id;
   }
 
-  remove(source, id) {
+  /**
+   * Removes a measurement and broadcasts the removed event.
+   *
+   * @param {string} id The measurement id
+   * @param {MeasurementSource} source The measurement source instance
+   * @return {string} A measurement id
```

Comment:
Doesn't appear to return anything

Reply:
I think I would switch it back to remove(source, id) since other APIs like getAnnotation are similarly ordered.

Reply:
If you check the function definition, source is not a required arg, its weird to use it in the first arg, and people will have to use remove(null, something) in order to call it without it.